src/app/learnocaml_common.ml

```diff
@@ -905,6 +905,8 @@ module Editor_button (E : Editor_info) = struct
       sync_exercise token id ~editor:(Ace.get_contents E.ace) >|= fun _save -> ()
 end
 
+
+
```

Comment:
nitpick: these extra newlines are unneeded I guess ;)

Reply:
Yes I just move a fonction in an other part of the documents and forgot to erase these lines